lib/jellyfish_web/api_spec/component/sip.ex

```diff
@@ -0,0 +1,84 @@
+defmodule JellyfishWeb.ApiSpec.Component.SIP do
+  @moduledoc false
+
+  require OpenApiSpex
+  alias OpenApiSpex.Schema
+
+  defmodule Credentials do
+    @moduledoc false
+
+    require OpenApiSpex
+
+    OpenApiSpex.schema(%{
+      title: "Credentials",
```

Comment:
I suggest we rename it to `SIPCredentials`, to indicate that those are sip-specific credentials.